### PR TITLE
Document KeycloakInstalled adapter

### PR DESCRIPTION
**Pull request by thomasdarimont**

### CODE REVIEW

**Review by cedosw — APPROVED** [review]
+1 docs would be great for this adapter

File: securing_apps/topics.adoc [hunk 1]
```diff
@@ -28,6 +28,7 @@ ifeval::[{project_community}==true]
 include::topics/oidc/java/servlet-filter-adapter.adoc[]
 include::topics/oidc/java/jaas.adoc[]
 endif::[]
+include::topics/oidc/java/installed-adapter.adoc[]
```
Comment (stianst) [inline securing_apps/topics.adoc]: Can you wrap this with:

    ifeval::[{project_community}==true] 
    ....
    endif::[]

**Comment by thomasdarimont** [comment]
Done.

**Comment by stianst** [comment]
Looks good to me. @patriot1burke since you've just looked at the KeycloakInstalled adapter could you take a quick look?

**Comment by stianst** [comment]
https://issues.jboss.org/browse/KEYCLOAK-5614

**Pull request merged**